sync/downstream.py

```diff
@@ -121,6 +121,12 @@ def pr_head(self):
         merged unless it happens to be a fast-forward"""
         return sync_commit.WptCommit(self.git_wpt, "origin/pr/%s" % self.pr)
 
+    @SyncProcess.error.setter
+    def error(self, value):
+        if value is not None and self.pr:
+            env.gh_wpt.add_labels(self.pr, "mozilla:gecko-blocked")
```

Comment:
We should also remove the label if its set and the value is `None`.

Reply:
The only problem I see there is in the case [here](https://github.com/mozilla/wpt-sync/blob/b15eb870b34f4e4e8d103f55d911c8046de68485/sync/downstream.py#L605) where `self.error = exception` and exception can be None. That doesn't mean that downstream has stopped failing if I understand it correctly.

We should probably have some sort of check that when all the try statuses are passing we can remove the label